gradle/libs.versions.toml

```diff
@@ -9,6 +9,7 @@ kotlinpoet = "1.14.2"
 mavenplugin = "0.25.3"
 moshi = "1.15.0"
 okhttp = "4.12.0"
+okhttp-junit = "5.0.0-alpha.11"
```

Comment:
lemme know if you'd rather keep the okhttp versions the same (and depend on the older `mockwebserver` artifact)

Reply:
Yeah let's keep them in sync.